scio-test/src/it/scala/com/spotify/ScioBenchmark.scala

```diff
@@ -260,16 +260,22 @@ object ScioBenchmark {
   private val K = 1000
   private val numPartitions = 100
 
-  private def randomUUIDs(sc: ScioContext, n: Long): SCollection[String] =
-    sc.parallelize(Seq.fill(numPartitions)(n / numPartitions))
+  final case class Elem[T](elem: T)
+
+  private def randomUUIDs(sc: ScioContext, n: Long): SCollection[Elem[String]] =
+    sc.parallelize((1 to numPartitions / 10).map(_ => Seq.fill(numPartitions)(n / numPartitions)))
```

Comment:
Instead of hard coding 10 here, should it be a multiple of `numWorkers` here? https://github.com/spotify/scio/pull/1290/files#diff-65c72767e390d7fa44ad2f4d472b681dR47